amundsen_application/static/images/favicons/dev/safari-pinned-tab.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
What's the use case for this file?

Reply:
I think this one is for the add to homescreen on safari mobile.

Reply:
Officially we don't support mobile for this tool -- not just from the Eng perspective but our designs also don't take mobile into account. I'd suggest removing any of the mobile specific values because we can say at this point we would never use them, but my understanding is that they all came from a generator? (in which case I'd just let the generator do it's thing and not mess with its output)

Reply:
Yeah, that is my take as well. We probably don't need half of this, but as it is just copy/paste from the generator output, it takes more work to prune it than to leave it here.